Add step to installation instruction

dastin-sandura opened the pull request:
- added step 0 to the installation instruction

dastin-sandura committed:
Add step to installation instruction

- added step 0 to the installation instruction

dastin-sandura committed:
Windows compatibility

- replace use of env variable PWD avaialble in unix with . avaialble in Unix and Windows

dastin-sandura commented on 1-resilient-messaging/docker-compose.yml:
```diff
@@ -4,7 +4,7 @@ services:
     image: simplycodedsoftware/php:8.2
     volumes:
-      - "$PWD:/data/app"
+      - ".:/data/app"
```
@dgafka $PWD doesn't work on Windows so I replaced it with .
Can you check if this version work on Mac/Linux?

dgafka replied:
Tak, będzie działać :)

Zmienisz też w drugiej częsci workshopu?

dastin-sandura replied:
Done

dastin-sandura committed:
Change port for ecotone_pulse

- change port from 3000 to 3001 due to error which is thrown when starting docker-compose with port 3000

dastin-sandura commented on 1-resilient-messaging/docker-compose.yml:
```diff
@@ -47,4 +47,4 @@ services:
       - default
     ports:
-      - "3000:80"
+      - "3001:80"
```
@dgafka when I did docker-compose up with port 3000 I received error:
`Error response from daemon: driver failed programming external connectivity on endpoint 1-resilient-messaging-ecotone_pulse-1 (4f6b0c987495941a3eb3b26da807a15a72d0eb8c087a6c19c7867e21d7267557): Bind for 0.0.0.0:3000 failed: port is already allocated`

dastin-sandura replied:
When I changed the port fro 3000 to 3001 after startup I can access Ecotone pulse in my browser at localhost:3001

dgafka replied:
Jak już odpowiadamy po polsku a i tak repo jest opisane po polsku, to tak będę kontynuował :P 

Z tego wynika, że masz już ten port zarezerwowany i coś Ci na nim chodzi.
To samo może być z bazą danych i RabbitMQ, które się bindują do host'a. 

To może warto zmienić wszystkie bindowania na porty 4000 i wyżej, tak aby nie wpaść w przypadkowe konflikty?

dastin-sandura replied:
Ok, przetestuję konfigurację z portami >4000 i dodam do tego pull request

dastin-sandura commented on 1-resilient-messaging/README.md:
```diff
@@ -8,6 +8,7 @@ W celu uruchomienia warsztatu będzie nam potrzebny jedynie [Docker](https://doc
 # Instalacja
 
+0. Otwórz konsolę komend i przejdź do folderu w którym jest ten plik README.md. Pamiętaj, że wszystkie komendy `docker-compose` zadziałają tylko wtedy, gdy wykonasz je będąc w folderze w którym istnieje plik docker-compose.yml
 1. Uruchom komendę `docker-compose pull && docker-compose up -d`
```
@dgafka co myślisz, żebyśmy w instrukcji zmienili komendę `docker-compose up `-d na komendę która pokaże wszystkie logi? Myślę, że może to ułatwić sprawdzanie dlaczego u kogoś może nie działać odpalenie projektu, jeśli takie przypadki się pojawią. W wersji -d, nie widać żadnych logów więc użytkownik nawet nie wie czy coś się odpaliło

dgafka replied:
Oki, poprawisz na `docker-compose pull && docker-compose up` ?

dastin-sandura committed:
Update commands for starting the application

 - remove the detached flag from starting up the docker compose for first and second workshop

dastin-sandura committed:
Windows compatibility - workshop 2

 - replace command line variable $PWD with solution compatible in Unix and Windows systems, the .

dastin-sandura commented on 1-resilient-messaging/README.md:
```diff
@@ -8,7 +8,8 @@ W celu uruchomienia warsztatu będzie nam potrzebny jedynie [Docker](https://doc
-1. Uruchom komendę `docker-compose pull && docker-compose up -d`
+0. Otwórz konsolę komend i przejdź do folderu w którym jest ten plik README.md. Pamiętaj, że wszystkie komendy `docker-compose` zadziałają tylko wtedy, gdy wykonasz je będąc w folderze w którym istnieje plik docker-compose.yml
+1. Uruchom komendę `docker-compose pull && docker-compose up`
 2. W momencie startu kontener z aplikacją zainstaluje dla nas wszystkie zależności. Można to sprawdzić przez `docker logs -f demo_development`
```
@dgafka uruchomienie tej komendy daje wynik Error: No such container: demo_development

Po wystartowaniu docker compose, widzę kilka kontenerów:

- 1-resilient-messaging-ecotone_pulse-1
- 1-resilient-messaging-rabbitmq-1
- ecotone_demo
- 1-resilient-messaging-database-1

Którego z nich powinniśmy podać w komendzie docker logs, żeby zobaczyć instalację zależności?

dgafka replied:
`ecotone_demo` instaluje zależności i na nim też odpalamy kodzik :)

dastin-sandura committed:
Dodaj info o Visual Studio Code

- dodaj sekcję z linkiem do pobrania VSCode i dodatkowo, link do rozszerzenia dla podpowiadania składni w PHP

dastin-sandura committed:
Update docker logs command

- put the correct container name into the command
